torch/csrc/lazy/core/shape_inference.cpp

```diff
@@ -674,46 +683,87 @@ std::vector<Shape> compute_shape_cast(const Output& input, const at::ScalarType&
 }
 
 
-// View Ops
+// View Ops (TODO kill these)
 std::vector<Shape> compute_shape_as_strided_view_update(const Output& target, const Output& input, const std::vector<int64_t>& size, const std::vector<int64_t>& stride, const int64_t& storage_offset) {
+  TORCH_INTERNAL_ASSERT(false, "view ops should never be called now that functionalization is being used");
```

Comment:
@bdhirsh Is there a way to disable the functionalization for ltc now? I felt like it might be easier for me to try to migrate to LTC first and then enable the functionalization. Otherwise doing two thing at once would be a bit tricky.

Reply:
Yeah, totally agreed - I think it would be ideal if we decouple "XLA integrating with LTC infra" with "XLA integration with functionalization", which I think should be possible.

We should be able to do that by leaving all of the existing LTC view infra intact. So before merging this PR, I'll remove these asserts. That way you can finish migrating onto LTC infra while re-using any of its view infrastructure, and we can kill that after once XLA is using functionalization (and has `view_copy` kernels implemented).